addons/mail/static/src/messaging/component/partner_mention_suggestion/partner_mention_suggestion.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="mail.messaging.component.PartnerMentionSuggestion" owl="1">
+        <a class="o_PartnerMentionSuggestion dropdown-item" t-att-class="{ 'active': props.isActive }" href="#" role="menuitem" t-att-data-id="partner.id"
```

Comment:
`active` from bootstrap is desired here? Otherwise `o-active` with custom css would be safer.